1-exercises/B-array-some/exercise.js

```diff
@@ -11,7 +11,11 @@ let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // If there is a null value in the array exit the program with the error code
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
-
+ for (let pair of pairsByIndex){
+  if (!pair){
```

Comment:
I think here it is better to use `pair === null` because that will check the type too